#Status message for the mesh subsystem
Header header
uint8 channel
uint8 layer
string own_mac
string parent_mac
int8 rssi
uint8 nodecount
uint32 heap
string[] children